### DISCUSSION TITLE
上次软件更新后变为免费版，重新授权不成功。

### DISCUSSION THREAD

**Discussion by shgzroo** [getlantern/lantern-forum-cn#864] [Q&A]
本着能从专用通道解决就不麻烦社区的原则，我首先通过软件界面上的报告问题选项提交了ticket，并于半月前收到如下邮件：
![mail](https://github.com/user-attachments/assets/0e8f6762-c5b8-48f8-b1d1-bf1b19f8faaa)
然后就没有然后了，ticket也已关闭。前阵想上helpdesk看看会否还有后续，竟发现同一邮箱注册的helpdesk账号也没了，密码找回也不成功。目前正设法使用免费流量来解决问题。
既然免费流量能保证一定速度，那授权不成功应该不是由连接问题导致的。
账号名为unnamedpku@gmail.com，方便的话请邮件联系，谢谢！

**Comment by MaxMKMax** [comment]
@shgzroo 您好. 请您也提供我支付单据, 我帮您提交问题上去.

**Reply by shgzroo** [reply]
您好，请问处理结果怎样了？

**Reply by shgzroo** [reply]
您好，请问处理结果怎样了？我需要重新上传截图吗？

**Reply by shgzroo** [reply]
![payment](https://github.com/user-attachments/assets/be66ef94-fd99-41ff-a97e-98e10cd6dac6)

**Reply by xiaoxin1982** [reply]
您好 我也有同样的问题，帮我调一下，前几天更新后就 登录不上，电脑也无法给手机授权登录，账号：21217694@qq.com
![Uploading 微信图片_20241013143109.jpg…]()

**Comment by greySkiesMarsh** [comment]
Hi @shgzroo 

I'll be more than happy to get the proper expiration date set for your plan if you're able to provide a screenshot or receipt of your original payment. In the meantime, I've upgraded your account to Pro for one month.

**Reply by shgzroo** [reply]
Hi, is the screenshot blocked by github? Do I need to reupload the screenshot?

**Reply by shgzroo** [reply]
![payment](https://github.com/user-attachments/assets/ba2d54a9-0e8e-4438-8cd1-fb91ed2a9ffd)

**Comment by shgzroo** [comment]
Hi,
I don't know if this would do the job...

greySkiesMarsh ***@***.***> 于2024年9月17日周二 03:02写道：

**Comment by shgzroo** [comment]
您好，这个截图够用吗？

Mike K ***@***.***> 于2024年9月17日周二 02:40写道：

**Comment by shgzroo** [comment]
@Mike777Khoo 您好，请问处理进度如何？我还需要等多久？之前延的一个月已经结束了。

**Comment by MaxMKMax** [comment]
@shgzroo 您好. 之前负责人已经解决问题了. 可能又有问题, 我帮您问看. 下周二给您答复. 谢谢.

**Comment by shgzroo** [comment]
@Mike777Khoo 辛苦。之前我得到了一个月的暂时性延期，但我似乎把github回复到邮箱的内容当成普通邮件回复了，导致截图发送不成功。上周我在上面重新发送了截图，不知现在您那能看到吗？

**Comment by shgzroo** [comment]
@Mike777Khoo 您好，现在是什么说法？

**Comment by tigersZ** [comment]


**Reply by tigersZ** [reply]
https://img.bian666.cf/file/1730521063680_1000013713.jpg


支付凭证截图

**Comment by shgzroo** [comment]
@Mike777Khoo 您好，又半个月过去了，请问答复是什么？

**Comment by shgzroo** [comment]
@Mike777Khoo 您好，请问答复是什么？问题一直没得到解决可以理解为穿墙不易，维护复杂；但迟迟不给答复是几个意思？

**Comment by shgzroo** [comment]
@Mike777Khoo 您好，已经一个多月了，上月19号所谓的“下周二给我”的答复在哪里？

**Comment by shgzroo** [comment]
@greySkiesMarsh Still no pro account after 4 months... No replies since 19th October... What has happened to you all?

**Comment by shgzroo** [comment]
@Mike777Khoo 您好，已经直奔两个月了，上上月19号所谓的“下周二给我”的答复在哪里？

**Comment by MaxMKMax** [comment]
@shgzroo 您好. 非常抱歉, 我以为您的问题已经被解决. 因为我已经帮您提交问题 3 次了. 刚好我们也换新的系统提交问题了. 我再次帮您提交问题. 然后看我可以帮您拿到补偿您 2 个月吗. 不好意思让您久等了. 谢谢您的非常耐心等待. 非常抱歉啊!!!

**Comment by MaxMKMax** [comment]
@shgzroo 我们也想知道问题出在哪里. 
您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by shgzroo** [reply]
根本连不上，点击发送之后长时间缓冲，图片也无法上传。

**Comment by MaxMKMax** [comment]
@shgzroo 同时您也试试看一个新的方法就是重新下载最新的版本然后重新登入看看.

**Comment by shgzroo** [comment]
@Mike777Khoo 已经重装不止一次了，仍然是免费版。考虑到九月中赠送的一个月专业版能马上到账号上，应该不是我这边本地的问题。另外，如本串最开头所述，我使用官方ticket提交系统的体验非常不好，我不觉得再提交一次能有助于问题的解决。

**Comment by shgzroo** [comment]
@Mike777Khoo 眼看就快半年了，去年8月续的两年，直到现在都没升级成功，具体出了什么情况？打算一直冷处理了？几年来一直两年两年的续，没出任何问题，就去年这么离谱...

**Comment by shgzroo** [comment]
@Mike777Khoo 之前使用所谓的官方ticket提交系统的体验见原帖，问题没解决不说账号也给我弄没了，你指望我再走一遍这个逼流程去和那几个不知道在干什么的机器人扯皮吗？

**Comment by MaxMKMax** [comment]
@shgzroo 您好. 不好意思啊让您久等了. 或许客服那里没留意到您的信息. 我这里马上帮您提交上去处理. 超级抱歉啊. 会尽快给您处理的!

**Comment by MaxMKMax** [comment]
@shgzroo 您好. 已经给您提交好了. 过几天请您留意一下邮箱. 谢谢您的耐心等待.

**Comment by shgzroo** [comment]
@Mike777Khoo 又一个月了，自从邮件称会有人来联络之后啥都没发生。依然是免费版。自从去年8月续费以来除了赠送的一个月外并没享受到任何服务。
![mail](https://github.com/user-attachments/assets/74cf83c1-f131-4ad5-a73c-584ee050c96b)
